### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by zadirion** [ssbc/patchbay#393]

**Comment by scocasso** [comment]
Not quite. I'd say it's dead in the water, but hasn't sunk yet.  Once in a while someone comes along and puts a little more air in it to keep it afloat.

**Comment by zadirion** [comment]
alrighty

**Comment by mathieujobin** [comment]
clicking the download link for this project lead to a release from 2019 ...
